DQM/SiPixelPhase1Heterogeneous/python/SiPixelPhase1HeterogenousDQM_FirstStep_cff.py

```diff
@@ -4,12 +4,11 @@
 from DQM.SiPixelPhase1Heterogeneous.siPixelPhase1MonitorRecHitsSoA_cfi import *
 
 from Configuration.ProcessModifiers.gpu_cff import gpu
-gpu.toModify(siPixelPhase1MonitorRecHitsSoA, pixelHitsSrc = "siPixelRecHitsPreSplittingSoA")
-
+from Configuration.ProcessModifiers.pixelNtupletFit_cff import pixelNtupletFit
+(gpu | pixelNtupletFit).toModify(siPixelPhase1MonitorRecHitsSoA, pixelHitsSrc = "siPixelRecHitsPreSplittingSoA")
```

Comment:
I have not tested it, but I think this should simply be
```suggestion
pixelNtupletFit.toModify(siPixelPhase1MonitorRecHitsSoA, pixelHitsSrc = "siPixelRecHitsPreSplittingSoA")
```
?

Reply:
Yes you are right, we never use the `gpu` alone for pixels.